### ISSUE TITLE
R Torch modules

### ISSUE THREAD

**Issue by jreps** [OHDSI/DeepPatientLevelPrediction#4]
- write generic code the will do the model fitting for any topology/type of model
- let user specify the optimization metric

**Comment by ted9219** [comment]
R also has a torch package [here](https://torch.mlverse.org/) and additional information is [here]( https://blog.rstudio.com/2020/09/29/torch/) based on libtorch (C++ backend) as same as PyTorch. I think it's different with the rTorch packages (binding PyTorch in python with reticulate env) Can we use this?

**Comment by ted9219** [comment]
I wrote some generic codes with Torch here #8 

I also felt like that torch package needs more instruction (there are only a few documents for description)

**Comment by ted9219** [comment]
Next steps
- Adding a dataloader function using the torch 
- Adding more variables for flexible NN settings
